product_sticker/models/product_attribute.py

```diff
@@ -0,0 +1,35 @@
+from odoo import api, fields, models
+
+
+class ProductAttribute(models.Model):
+    _inherit = "product.attribute"
+
+    sticker_ids = fields.One2many(
+        comodel_name="product.sticker",
+        inverse_name="product_attribute_id",
+        string="Product Stickers",
+        domain=lambda s: [
+            "|",
+            ("company_id", "=", False),
+            (
+                "company_id",
+                "in",
+                s.env.context.get("allowed_company_ids", s.env.company.ids),
+            ),
+        ],
+    )
+    sticker_count = fields.Integer(
+        compute="_compute_sticker_count",
+        store=False,
+    )
+
+    @api.depends("sticker_ids")
+    def _compute_sticker_count(self):
+        for attribute in self:
+            attribute.sticker_count = len(
+                attribute.sudo().sticker_ids.filtered(
+                    lambda i: not i.company_id
+                    or i.company_id.id
+                    in self.env.context.get("allowed_company_ids", self.env.company.ids)
+                )
+            )
```

Comment:
IMHO all this code should bo in an `AbstractModel` mixin. Then, apply the mixin to `product.attribute` and `product.attribute.value`.

Reply:
The `inverse_name` on `sticker_ids` fields doesn't allow to put it on a mixin.